src/Bicep.Core/Emit/EmitLimitationCalculator.cs

```diff
@@ -48,6 +48,8 @@ public static EmitLimitationInfo Calculate(SemanticModel model)
             BlockModuleOutputResourcePropertyAccess(model, diagnostics);
             BlockSafeDereferenceOfModuleOrResourceCollectionMember(model, diagnostics);
             BlockCyclicAggregateTypeReferences(model, diagnostics);
+            BlockUserDefinedFunctionsWithoutExperimentalFeaure(model, diagnostics);
```

Comment:
>BlockUserDefinedFunctionsWithoutExperimentalFeaure [](http://example.com/codeflow?start=12&length=50)

This is certainly a convenient place to put these checks, but I wonder if we need to rename this class. (Blocking UDFs if the experimental feature is disabled isn't really due to JSON limitations.)

Reply:
Good point, it's a convenient place to put any sort of ad hoc check. Will raise an issue to rename it.